app/controllers/admin/bulk_assignments_controller.rb

```diff
@@ -18,7 +18,7 @@ def enqueue_assignments
       flash[:success] = success_message
       redirect_to admin_split_path(@split)
     else
-      flash[:error] = "Please address the errors below."
+      flash[:danger] = "Please address the errors below."
```

Comment:
Primer uses `:danger` rather than`:error` as the flash type for form errors.

Reply:
I remember there being some issue in other apps (maybe `clinic`?) where some framework code was emitting a flash message using a key that was incompatible with our frontend. Maybe it was `devise` or omniauth-related? Worth spot-checking that failure messages from some of the implicitly mounted rails engines will still show up properly

Reply:
Rather than change these setters, I created a FlashComponent that will map the flash names to the `scheme` values expected by the Primer Banner component.